spec/features/maneuver_finding_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+describe 'finding a participant' do
+  let!(:maneuver) { create :maneuver }
+  let!(:record_1) do
+    create :maneuver_participant,
+           :perfect_score, maneuver: maneuver
+  end
+  let!(:record_2) do
+    create :maneuver_participant,
+           :perfect_score, maneuver: maneuver
+  end
+  let!(:record_3) do
+    create :maneuver_participant,
+           :perfect_score, maneuver: maneuver
+  end
+  context 'finds the previous' do
+    it 'at first participant gets first participant' do
+      when_current_user_is :admin
+      visit maneuver_participant_path(record_1.id)
+      click_link 'Previous participant'
+      expects = 'This is the first participant who completed this maneuver.'
```

Comment:
`expects` is a kind of odd variable name here - previous PRs have gone for `expected`.